webpack.config.dev.js

```diff
@@ -0,0 +1,54 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+module.exports = {
+	entry: './src/index.jsx',
+	output: {
+		path: path.resolve(__dirname, 'dist'),
+		filename: 'bundle.js',
+	},
+	resolve: {
+		extensions: ['.js', '.jsx']
+	},
+
+	watch: true,
+	mode: 'development',
+	devtool: 'source-map',
+
+	module: {
+		rules: [
+			{
+				test: /\.js$/,
+				use: 'babel-loader',
+				exclude: /node_modules/,
+			},
+			{
+				test: /\.jsx?$/,
+				use: 'babel-loader',
+			},
```

Comment:
Probably, you could replace these two rules for jsx and js on like this:
`{
      test: /\.(js|jsx)$/,
      exclude: /node_modules/,
      loader: 'babel-loader'
  },`